scripts/test-setup.rs

```diff
@@ -0,0 +1,29 @@
+use std::env;
+use std::time::Duration;
+
+use momento::config::configurations;
+use momento::{CacheClient, CredentialProviderBuilder};
+
+#[tokio::main]
+async fn main() {
+    let cache_name =
+        env::var("TEST_CACHE_NAME").expect("environment variable TEST_CACHE_NAME should be set");
+
+    let credential_provider =
+        CredentialProviderBuilder::from_environment_variable("TEST_API_KEY".to_string())
```

Comment:
supernit: i've been slowly trying to standardize on MOMENTO_API_KEY for these. In the future I can imagine that we might have a default credentialprovider that looks for creds in a few well known places, including a well known env var name, like AWS does. I don't see any super compelling reason to make tests use a different env var name. Thoughts?

Reply:
I'm good with MOMENTO_API_KEY. I agree that there isn't much reason to separate them, other than if someone consistently needed to run a local application using a different key than they were using to develop an sdk.

Reply:
thx